2017/hipchat-breakin.md

```diff
@@ -0,0 +1,240 @@
+
+Wenn man nun eigene Infrastruktur betreibt und die Daten der Nutzer die eigenen
+Server nicht verlassen, können diese auch nicht verkauft oder missbraucht werden.
+Und dafür lohnt sich die Arbeit! Wer will schon eigene
+[private Bilder frei im Netz][fappening] kursieren sehen?
+Oder [Passwort-Datenbanken][leak-linkedin]? Oder
+[gekaperte E-Mail-Konten][yahoo-hack]?
+
+## Wer ist nun schuld? Der Nutzer oder der Anbieter?
+Zu gerne würde man an dieser Stelle den schlechten Zustand unserer Technologie
+auf die Konsumenten schieben. Denn am Ende halten sich auf dem Markt nur
+Produkte, die von Kunden auch nachgefragt werden. Und diese sind zur Zeit nun
+mal überwiegend datenschutzfeindlich. Es geht sogar so weit, dass die großen
+Plattformen [mehr und mehr an politischem Gewicht gewinnen][denmark-digital-ambassador].
+Und das alles, obwohl seit Jahren davor gewarnt wird, welche Macht
+[die Anhäufung von Daten][vds-spitz] haben kann!
+
+„Halt, stopp!“, rufen Menschen denen man solche Dinge an den Kopf wirft – zu Recht!
+Diese Vorwürfe sind natürlich haltlos. **Die Abwelzung von Verantwortung ist
+nämlich das eigentliche Problem.** Klar ist: jeder Mensch, der Plattformen wie
+Facebook, SaaS wie Slack oder
+Smartphone-Apps wie WhatsApp nutzt muss lernen welche Folgen dies
+beinhaltet. Es ist im 21. Jahrhundert einfach nicht mehr akzeptabel, als Bürger
+einer Demokratie ein technisch ungebildeter Konsument zu sein.
+Und das Problem wächst, je
+technologisierter unser Alltag und unsere Arbeitswelt wird.
+
+Aber wer steht nun in der Pflicht, Technologien gesellschaftlich akzeptabel und
+nachhaltig zu gestalten?
+
+Erstens: der Konsument bzw. Kunde eines Softwareprodukts. **Du machst durch deine
+Wahl einen Unterschied, welche Software auf den Markt kommt und welche bleibt**. An
+Beispielen wie MySpace und StudiVZ sieht man, dass auch große soziale Netzwerke
+ihre Nutzer wieder verlieren können. Und sogar zu Facebook gibt es immer wieder
+Meldungen, in denen [eine Verringerung der Nutzerzahlen][fb-decline]
+prognostiziert wird.
+
+Zweitens: der Anbieter und die Entwickler hinter einem Produkt! **Alle Menschen der
+Softwareindustrie tragen [eine Verantwortung][iug]**. Jeder Schnippsel Code verändert
+die Softwarelandschaft zum positiven oder negativen. Es ist im 21. Jahrhundert
+einfach nicht mehr akzeptabel, dass Programmierer irgendwelche Tools auf Github
+veröffentlichen, die dann in Monopol- oder Militärsoftware integriert werden!
+Auch hier ist noch ein riesen Paket Aufklärungsarbeit zu leisten, um
+Goldrauschprogrammierern ihrer Mitverantwortung bewusst zu machen.
+
+Die Informatik muss sich öffnen und aus der Blase hinaustreten. Es ist schön zu einem kleinen
+Kreis Eingeweihter zu gehören und mit dem eigenen Wissen viel Geld machen zu
+können – langfristig gesehen ist es aber fragwürdig, ob wir
+[Algorithmen, die von einem homogenen Bruchteil der Bevölkerung entworfen wurden][ai-bias],
+gesellschaftliche Prozesse anvertrauen wollen!
+
```

Comment:
1. würde ich weniger Ausrufzeichen verwenden, also hier weglassen.
2. Das AI-Problem ist ja nur eins. Das Sammeln und zentrale Vorhalten von Daten an sich ist ja schon problematisch. Die Botschaft könnte hier sein: ohne eine halbwegs funktionierende Gesellschaft sind IT-Kenntnisse überflüssig und wertlos